presto-main/src/main/java/io/prestosql/operator/TaskContext.java

```diff
@@ -368,6 +373,14 @@ public Duration getFullGcTime()
         return new Duration(max(0, endFullGcTimeNanos - startFullGcTimeNanos), NANOSECONDS);
     }
 
+    /**
+     * verifies how many objects were moved to the Garbage Collector.
```

Comment:
This method isn't verifying anything. It's just returning the calculated value. So the description should be:
"Returns how many objects were moved to the Garbage Collector in the task context."